hack/seed-catalog-items.sh

```
@@ -0,0 +1,225 @@
+        ((SKIPPED++))
+        return 0
+    fi
+
+    local response
+    response=$(curl -sf -X POST \
+        -H "Authorization: Bearer ${TOKEN}" \
+        -H "Content-Type: application/json" \
+        -d "$payload" \
+        "${API_URL}/api/private/v1/${type}" 2>&1)
+
+    if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)